flake.nix

```diff
@@ -1,75 +1,38 @@
 {
-  description = "Agoric SDK Development Environment";
+  description = "Agoric SDK";
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    flake-utils.url = "github:numtide/flake-utils";
+    unstable-nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    # Track a separate nixpkgs for JS/TS toolchains
+    nixpkgs-js.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
help me understand... why should we bind 3 different attributes to the same github url?